meetings/2017-05-10.md

```diff
@@ -0,0 +1,186 @@
+
+@ofrobots: Very little cost to add --debug-brk back as an alias, and given so much existing documentation, makes sense to add back in.
+
+@jasnell: Makes sense, adding back as alias is cheap.
+
+@trott: Ben and Jeremiah have objections, need to address them or call for a vote.
+
+@Fishrock123: Why is --debug-brk needed in a later version? A: For consistency for debugging tools.
+
+@joshgav: Change to remove --debug-brk will happen at a version boundary (8.0.0) so can version detection logic just differentiate between 7.x and 8.x?
+
+@jasnell: Would be best to have --debug\* flags as aliases for now (8.0.0), could print a deprecation message asking to use --inspect\*.
+
+Proposal: Restore --debug-brk as an alias that also prints deprecation warnings.
+
+@ChALkeR: but actual behavior of --debug-brk will have changed, so we should just print a message and exit.
+
+@jasnell: In that case should be an error.
+
+@ChALkeR: Will investigate if alias would break anything.
+
+**Next steps**
+
+* Никита to review use as alias for potential problems.
+* If/when Никита approves: restore --debug-brk as an alias to --inspect-brk that also prints a deprecation warning.
+
+---
+
+### Upgrade openssl-1.1.0e for Node-v8 (DO NOT LAND THIS) [node#11828](https://github.com/nodejs/node/pull/11828)
+
+@mhdawson to review with @sam-github to determine next steps.
+
+---
+
+### Invert dependency between core and readable-stream [node-eps#49](https://github.com/nodejs/node-eps/pull/49)
+
+Out of time and this won't ship in 8.0.0, so postpone discussion.
+
+---
+
+### Update on 8.0.0
+
+@jasnell: Significant perf regression with EventEmitter and streams, perhaps related to V8 changes, James and V8 team are investigating.
+
+@mcollina: Crankshaft added an unoptimized path for Array.shift, need to backport fix from later V8 versions.
+
+@mhdawson: Should we add RC's to benchmarks? A: Yes.
+
+@jasnell: Preparing a Release Candidate today, but still might be more breaking changes. Please test!
+
+* PR for npm@5 opened (<https://github.com/nodejs/node/pull/12936>). Current plan is to include if it's stable.
+
+## Q&A, Other
+
+* Q: How to contribute to Node.js?
+
+	* http://nodetodo.org.
+	* Code and Learn events, there will be one in Shanghai this summer as part of JSConf China.
+
+* Take a look at WG charter recommendations from James.
+
+* Seeking review and feedback on <https://github.com/Microsoft/node-diagnostic-channel>, a proposal and implementation for sharing module patches and perhaps eventually not needing them.
+
+## Upcoming Meetings
+
+* **Node.js Foundation Calendar**: <https://calendar.google.com/calendar/embed?src=nodejs.org_nr77ama8p7d7f9ajrpnu506c98%40group.calendar.google.com>
+
+Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.
+
+* **TC39 Meetup**: May 22,
+  [CTC#67](https://github.com/nodejs/CTC/issues/67)        
+* **CTC**: May 17, 2000 UTC
```

Comment:
Yes, that's right. Would it be more clear if we started writing it `20:00` instead? Also I'm never sure if it should be `UTC 20:00` or `20:00 UTC`. Not sure if it even matters.